src/main/webapp/webpack.config.dev.js

```diff
@@ -1,5 +1,5 @@
 exports.config = {
-  mode: "none",
+  mode: "development",
```

Comment:
What's this addition about?

Reply:
This was causing me an issue with running the watch statement for compiling JS files.  Just added here so that I could have it working.